composer/modules/js-tests/src/test/resources/failing/natives-9.bal

```diff
@@ -4,63 +4,93 @@ package ballerina.io;
+@Param {value:"content: Text content which should be written"}
+@Param {value:"startOffset: If the content needs to be written with an offset, the value of that offset"}
+@Return { value:"Number of characters written"}
 public native function <CharacterChannel channel> writeCharacters(string content,int startOffset)(int);
 
-@Description {value:"Function to close character channel"}
+@Description {value:"Function to close a character channel"}
+@Param {value:"channel: The CharacterChannel to be closed"}
 public native function <CharacterChannel channel> closeCharacterChannel();
 
 @Description { value:"Function to read bytes"}
-@Param { value:"numberOfBytes: number of bytes which should be read" }
-@Return { value:"blob: the bytes which were read" }
-@Return {value:"int: number of bytes which were read"}
+@Param {value:"channel: The ByteChannel to read bytes from"}
+@Param { value:"numberOfBytes: Number of bytes which should be read" }
+@Return { value:"The bytes which were read" }
+@Return { value:"Number of bytes read"}
 public native function <ByteChannel channel> readBytes (int numberOfBytes) (blob, int);
 
+@Description {value:"Function to read all bytes in the given I/O source"}
+@Return {value:"all bytes read from the channel"}
+public native function <ByteChannel channel> readAllBytes()(blob,int);
+
 @Description { value:"Function to write bytes"}
-@Param { value:"content: bytes which should be written" }
-@Param { value:"content: whether the bytes should be writtne with an offset" }
-@Return {value:"int: number of bytes written"}
+@Param {value:"channel: The ByteChannel to write bytes to"}
+@Param { value:"content: Bytes which should be written" }
+@Param { value:"startOffset: If the bytes need to be written with an offset, the value of that offset" }
+@Return { value:"Number of bytes written"}
 public native function <ByteChannel channel> writeBytes(blob content,int startOffset) (int);
 
-@Description { value:"Function to close the byte channel"}
+@Description { value:"Function to close a byte channel"}
+@Param {value:"channel: The ByteChannel to be closed"}
 public native function <ByteChannel channel> close();
+
+@Description {value:"Function to check whether next record is available or not"}
+@Param {value:"channel: The DelimitedRecordChannel to read text records from"}
+@Return {value:"True if the channel has more records; false otherwise"}
+public native function <DelimitedRecordChannel channel> hasNextTextRecord () (boolean);
+
+@Description {value:"Function to return a ByteChannel related to the file. This ByteChannel can then be used to read/write from/to the file."}
+@Param {value:"path: The path of the file"}
+@Param {value:"accessMode: Specifies whether the file should be opened for reading or writing (r/w)"}
+@Return {value:"ByteChannel which will allow to perform I/O operations"}
+public native function openFile (string path, string accessMode) (ByteChannel);
```

Comment:
Shall we leave an empty line here ?

Reply:
eeb1d11ba2a2eb0836598e496bd0f00602d5acf9